WordPress/Jetpack/Classes/ViewRelated/WordPress-to-Jetpack Migration/Common/MigrationAppDetection.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+enum WordPressInstallationState {
+    case wordPressNotInstalled
+    case wordPressInstalledNotMigratable
+    case wordPressInstalledAndMigratable
+}
+
+struct MigrationAppDetection {
+
+    static func getWordPressInstallationState() -> WordPressInstallationState {
+        if UIApplication.shared.canOpen(app: .wordpressMigrationV1) {
+            return .wordPressInstalledAndMigratable
+        }
+
+        if UIApplication.shared.canOpen(app: .wordpress) {
```

Comment:
⚠️ This will be `false` if WordPress is installed but not a release build. Instead of checking for all possible schemes I decided to only support release builds since that's what the App Store and TestFlight use and `wordpressmigration+v1` works across all build types.